cornucopiaCards/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "cornucopia-cards",
+  "version": "1.0.0",
+  "description": "",
```

Comment:
We need a proper description and maybe remove the "main"-field, as this file is not present